cli-output/src/cli_output.rs

```diff
@@ -277,21 +281,29 @@ impl fmt::Display for CliEpochInfo {
                 remaining_slots_in_epoch
             ),
         )?;
+        let time_elapsed = if let (Some(start_block_time), Some(current_block_time)) =
+            (self.start_block_time, self.current_block_time)
+        {
+            Duration::from_secs((current_block_time - start_block_time) as u64)
+        } else {
```

Comment:
Should we annotate the fallback case somehow? Maybe `(imprecise)`, not that either is particularly "precise" :sweat_smile:

Reply:
Yeah, some kind of annotation is a good idea. I'll think on a good term....

Reply:
@t-nelson what do you think of https://github.com/solana-labs/solana/pull/18258/commits/9e5f96c6bd1adec4da68b1ee6d32d1b73d0c0c7e ? I tried a bunch of one-word tags -- before everything, after everything, or by the affected data -- but didn't find anything that was both clear and non-disruptive to the `Epoch Completed Time:` line (which is already complex, due to parenthetical)

Reply:
Works for me!

NIT: Maybe an extra newline in front of the annotation to make it stand out?